core/src/main/java/org/elasticsearch/rest/action/admin/indices/RestIndicesStatsAction.java

```diff
@@ -157,7 +158,7 @@ public boolean canTripCircuitBreaker() {
         return false;
     }
 
-    private static final Set<String> RESPONSE_PARAMS = Collections.singleton("level");
+    private static final Set<String> RESPONSE_PARAMS = new HashSet<String>(Arrays.asList("level", "include_segment_file_sizes"));
```

Comment:
Gotcha. I see some examples of that now using the `request.paramAsBoolean`.  I'll correct it this way, and correct it in the node stats, too.